src/ResourceManagement/ResourceManager/Core/ExternalChildResourceCollection.cs

```diff
@@ -69,12 +69,15 @@ public Task<List<FluentModelTImpl>> CommitAndGetAllAsync(CancellationToken cance
                                 exceptions.Add(deleteTask.Exception);
                             }
                         }
+                        if (deleteTask.IsCanceled)
```

Comment:
curious, why specific to deleteTask

Reply:
Before execution, we will label all pending child tasks with `ToDelete`/`ToCreate`/`ToUpdate`. And this situation may happen at any of the pending tasks, not only limited by one kind of them.